sdk/src/rest/commands/server/info.ts

```diff
@@ -5,6 +5,7 @@ export type ServerInfoOutput = {
 	project: {
 		project_name: string;
 		default_language: string;
+		is_public_registration_enabled: boolean; // TODO Get opinion from @Brainslug
```

Comment:
This should be sufficient IMO, the type already indicates that it's a toggle state:
```suggestion
		public_registration: boolean;
```

E.g. comparable to states in Collection metadata:
https://github.com/directus/directus/blob/d10385d30305adbe1b9abfa4bdefcff1836effca/packages/system-data/src/types.ts#L26-L33